### ISSUE TITLE
Crest

### ISSUE THREAD

**Issue by tudorgroza** [obophenotype/human-phenotype-ontology#6090]
**Preferred term label:**
Crest

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@tudorgroza  Sorry, but could you provide some more context? I do not know what crest is referring to here!

**Comment by tudorgroza** [comment]
@pnrobinson - Apologies for the lack of context. All terms I've submitted are in the context of rare pulmonary disorders - in particular the various sub-types of pulmonary hypertension. One of the key features of the group is CREST syndrome (https://en.wikipedia.org/wiki/CREST_syndrome). This does exist as a disease definition in Orphanet - hence not sure to what extent it makes sense as a phenotype. Thought I'd ask nevertheless. Thank you.

**Comment by pnrobinson** [comment]
I see - we would keep CREST as a disease (e.g. https://hpo.jax.org/app/browse/disease/ORPHA:90290). HPO terms should be for specific abnormalities, not for entire disease entities!
